Sidney-/slugtrade
# Wishlist

New Functionality:
- There should be a wishlist at the bottom of the profile page
  - It should have a text field to enter wishlist items into and an 'add' button
  - When you enter text into the field and hit enter or click add, an unordered list should be populated 
     with your new item
  - After you enter a new item, the page should refresh and auto scroll back down to the wishlist
  - Each item already in the wishlist should have a button with an X next to it for deleting the item
  - deleting items should not refresh the page
- On other user's profiles, you should be able to see their wishlist, but not edit it
- If you navigate to your own profile by selecting your name from the /users endpoint, all of the above
  functionality should still work

How to test:
1) Go to /profile and scroll down to the bottom of the page to ensure that the wishlist is there
2) Enter items into the wishlist and make sure it is populated correctly
3) Delete items from the wishlist and ensure that they get deleted (and stay deleted after a refresh)
4) Navigate to your "arbitrary" profile by clicking on it from the /users endpoint (the new profile functionality that Shane just checked in)...then run tests 1-3 on that page, make sure they still run
5) Switch users (i.e., if you were on admin, log in as someone else) then view the other profile that you were logged in as 
   - Make sure that you can see but not edit the wishlist, correctly filled in.
